<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolyLineTool</class>
 <widget class="QWidget" name="PolyLineTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="activeHLayout">
    <item>
     <widget class="QLabel" name="activeLabel">
      <property name="text">
       <string>Active:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="activeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="resetPLButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Polyline Tool</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="insidePLGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Inside Polyline</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="insideVLayout">
     <item>
      <widget class="QPushButton" name="insideFillButton">
       <property name="toolTip">
        <string>Flip across the axial plain</string>
       </property>
       <property name="text">
        <string>Fill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insideEraseButton">
       <property name="toolTip">
        <string>Flip across the coronal plain</string>
       </property>
       <property name="text">
        <string>Erase</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
